config/migrations/20200923144227-notule-cleanup1.sparql

```diff
@@ -0,0 +1,23 @@
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX brc: <http://kanselarij.vo.data.gift/id/concept/beslissings-resultaat-codes/>
+PREFIX type: <http://kanselarij.vo.data.gift/id/concept/document-type-codes/>
+
+DELETE {
+  GRAPH ?g {
+       ?notule ext:getekendeDocumentVersiesVoorNotulen ?doc.
+  }
```

Comment:
`?zitting ext:algemeneNotulen ?notule.` ook hier ?
Dan is de relatie tussen de zitting en notule opgekuist.

Reply:
Die gebeurd 1 query verder